safe/report/map.py

```diff
@@ -88,6 +90,15 @@ def set_impact_layer(self, layer):
         """
         self.layer = layer
 
+    def set_legend_layers(self, legend_layers):
+        """Set the layers that will be shown in the legend.
+        .. note: Added in 2.2 to cater the obsolescence of legend.model() in
+        QGIS 2.6. The layers must be an instance of QgsLayerTreeGroup.
+        :param legend_layers: The layers to be shown in the legend.
+        :type legend_layers: QgsLayerTreeGroup
+        """
+        self.legend_layers = legend_layers
+
     def set_north_arrow_image(self, north_arrow_path):
         """Set image that will be used as organisation logo in reports.
```

Comment:
Don't bother with this. I am going to remote safe.report.map. I just need to change its usages using new class